shared/people/follow-notification/index.js

```diff
@@ -31,23 +40,25 @@ export const FollowNotification = (props: Props) => {
   }
   const username = props.newFollows[0].username
   return (
-    <PeopleItem
-      badged={props.badged}
-      icon={
-        <Avatar username={username} onClick={() => props.onClickUser(username)} size={isMobile ? 48 : 32} />
-      }
-      when={props.notificationTime}
-      contentStyle={{justifyContent: 'center'}}
-    >
-      <Text type="Body" style={{marginTop: 2}}>
-        <ConnectedUsernames
-          {...connectedUsernamesProps}
-          usernames={[username]}
-          onUsernameClicked={props.onClickUser}
-        />{' '}
-        followed you.
-      </Text>
-    </PeopleItem>
+    <ClickableBox onClick={() => props.onClickUser(username)}>
+      <PeopleItem
+        badged={props.badged}
+        icon={
+          <Avatar username={username} onClick={() => props.onClickUser(username)} size={isMobile ? 48 : 32} />
```

Comment:
Don't need onClickUser here anymore (in connectedUsernames I think you do because it defaults to opening the tracker on desktop)

Reply:
Tested on desktop and it's still needed, not sure why. :)